cmd/csi-proxy/main.go

```diff
@@ -68,7 +70,12 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	s := server.NewServer(apiGroups...)
+
+	enableMetrics := *metricsBindAddr != ""
+	if enableMetrics {
+		metrics.ExportMetrics(*metricsBindAddr)
```

Comment:
If I want to enable metrics and by some reason `ExportMetrics` fails then this not fail and continue silently.

What about `metrics.ExportMetrics` returning an error that's checked while setting up CSI Proxy? I think we should panic with the error returned.